src/components/ComfirmationModal/ConfirmationModal.spec.js

```diff
@@ -0,0 +1,33 @@
+import {render, screen} from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import ConfirmationModal from './ConfirmationModal';
+
+describe('Confirmation Modal', () => {
+  it('renders correctly', () => {
+    render(<ConfirmationModal />);
+
+    screen.getByText('Do you really want to delete this Activity?');
+
+    screen.getByRole('button', {name: 'Cancel'});
+    screen.getByRole('button', {name: 'Delete'});
+  });
```

Comment:
This doesn't test anything. The expect is missing here.

Reply:
Just talked to the other coaches about it. And since it is with `getByRole` it does throw an error if the element is not found. 

I prefer to use `expect` but this is not wrong, sorry about that :)